lib/post_message_definition.yml

```diff
@@ -16,6 +16,14 @@ post_messages:
       ping:
         payload:
           <<: *user_session_properties
+      navigation:
```

Comment:
Let's keep this post message out of the docs like we discussed.

Reply:
Since this is part of the 'generic' events, it doesn't get added to the docs, but I can add the `documented: false` to it, just so if those ever get added, this one stays out and we don't need to remember to update it.

Reply:
Yeah I'm sure we'll forget to do that in the future if we don't do it now.

Reply:
Updated!